mod/calculated_metrics/set/type_plus_right/metric/formula/descendant.rb

```diff
@@ -1,7 +1,7 @@
 
 def inheritance_formula
   item_names.map do |item|
-    "{{#{item}}}"
+    "{{#{item} | not_researched: false}}"
```

Comment:
I decided to go for "false". Maybe more intuitive in a boolean expression. But nil should work, too.